test/e2e/network/conntrack.go

```diff
@@ -138,15 +153,17 @@ var _ = SIGDescribe("Conntrack", func() {
 		err = validateEndpointsPorts(cs, ns, serviceName, portsByPodName{podBackend1: {80}})
 		framework.ExpectNoError(err, "failed to validate endpoints for service %s in namespace: %s", serviceName, ns)
 
-		// Check that the pod receives the traffic
-		// UDP conntrack entries timeout is 30 sec by default
+		// Note that the fact that Endpoints object already exists, does NOT mean
+		// that iptables (or whatever else is used) was already programmed.
+		// Additionally take into account that UDP conntract entries timeout is
+		// 30 seconds by default.
+		// Based on the above check if the pod receives the traffic.
 		ginkgo.By("checking client pod connected to the backend 1 on Node IP " + serverNodeInfo.nodeIP)
-		time.Sleep(30 * time.Second)
-		logs, err = e2epod.GetPodLogs(cs, ns, podClient, podClient)
-		framework.ExpectNoError(err)
-		framework.Logf("Pod client logs: %s", logs)
-		if !strings.Contains(string(logs), podBackend1) {
-			framework.Failf("Failed to connecto to backend 1")
+		if err := wait.PollImmediate(5*time.Second, time.Minute, logContainsFn(podBackend1)); err != nil {
```

Comment:
maybe we can relax the interval, I mean, if it connects it will show up in the logs.
Anyway, you know better than me and anybody about this, just trying to reduce load in the e2e tests.

Reply:
IIUIC polling just improve the speed of the test only, it should be enough changing the sleep to 1 minute

Reply:
Yes - we can change to sleep 1m, but if polling logs every 5s can overload us, we have bigger problems...

Reply:
well, the kube-cli tests take more than 20 secs to attach to  stdin sometimes,

Reply:
I don't think we attach to get logs